source/docs/articles/local/cli/machine-tokens.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Creating and Revoking Machine Tokens
+description: Learn how to create a machine token in order to use Terminus.
+---
+
+Machine tokens are used to uniquely identify your machine and securely authenticate via Terminus (as of the 0.11.0 release).
```

Comment:
Right now we should recommend that people use master for this. 

Let's put this in an alert box. Once the release is cut we can update.